qemu/tests/live_backup_base.py

```diff
@@ -139,6 +142,22 @@ def backup_check(self, compare_image=None):
             source_image = qemu_storage.storage.get_image_filename(params, data_dir)
             qemu_image.compare_images(compare_image, source_image)
 
+    def get_image_size(self, image):
+        """
+        Get image size for given image.
+        :param image: image file.
+        :return: image size.
+        """
+        params = self.params.object_params(image)
+        qemu_image = qemu_storage.QemuImg(params, self.data_dir, image)
+        image_size = 0
```

Comment:
This should not be `0` if the really bad thing happened. Instead, I suggest raising the error directly.